akka-stream/src/main/mima-filters/2.5.x.backwards.excludes/issue-21753-sslconfig.excludes

```diff
@@ -0,0 +1,12 @@
+# #21753 internal method WithSSLEngine renamed and changed
```

Comment:
I was briefly worried that we might use those in akka-http but we don't. Seems Artery is the only place they are used. Good to remove a bit of internal API!